cdap-ui/server/token.mjs

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright © 2019 Cask Data, Inc.
+ * Copyright © 2019-2020 Cask Data, Inc.
  *
  * Licensed under the Apache License, Version 2.0 (the "License"); you may not
```

Comment:
Is there a specific reason why we renamed only the token and token_test to mjs modules? The other serverside files seem to be still having the .js extension. Is it solely for the token_test to be run locally?

Reply:
yes, it's solely because we need to run token_test, as token_test is not built. We can rename everything to .mjs, but at the moment there's no specific need to do that